src/phoenix/core/model_schema.py

```diff
@@ -43,6 +43,8 @@
 from typing_extensions import TypeAlias, TypeGuard
 from wrapt import ObjectProxy
 
+from phoenix.config import GENERATED_DATASET_NAME_PREFIX
```

Comment:
@RogerHYang this slightly breaks the encapsulation that you achieved in `core` - wasn't sure if you wanted to preserve that and make `display_name` be passed as a parameter.